### ISSUE TITLE
A11y_AzureDataStudio_Dashboard_screen Reader:While giving Enter hide text label button, NVDA is announcing Show text label twice

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#22815]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Win32; #WCAG1.3.1; #Info and Relationships; #NVDA; #Screen Reader; #Benchmark; #AILimited; #Element:Button; 

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
NVDA Version 2023.1
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server connection. 
4. Then select 'Manage' from the context menu to view the dashboard.
5.Tab till hide text label button
6.Observe  when we give enter on hide text label , Show text label is announcing twice or not
### Actual Result:
While giving Enter hide text label button, NVDA is announcing Show text label twice 

**Notes**

1. No issue with JAWS
2. Issue Repro for collapse widget button also

### Expected Result:
NVDA should not announce twice the same information.
### Attachment:
**NVDA**
![Screenshot (71)](https://user-images.githubusercontent.com/100189407/229530656-2d9a1158-978c-4002-ae21-04c728014ff8.png)
![Screenshot (72)](https://user-images.githubusercontent.com/100189407/229530951-fd5dc87e-ea4a-4614-8a9e-22a57ec41918.png)
**JAWS**
![jaws no repro 1](https://user-images.githubusercontent.com/100189407/229709382-a2f9e4f0-1d0e-4930-be79-f184b6a3d46f.png)
![Screenshot (81)](https://user-images.githubusercontent.com/100189407/229712240-c67d2565-b00c-4aae-bacd-49b7dda607f6.png)

**Comment by VijayaBharathyA** [comment]
Primary Bug :https://github.com/microsoft/azuredatastudio-mysql/issues/189
